doc/api-specification.yaml

```diff
@@ -142,6 +141,8 @@ paths:
                 $ref: '#/definitions/Comment/properties/body'
               published:
                 $ref: '#/definitions/Comment/properties/published'
+              dependency_id:
```

Comment:
This should be in a dedicated PR, it is not related with the URL nesting. Same thing for the `summary` and `description` few lines above.